dataframe/dataframe.go

```diff
@@ -1037,21 +1056,66 @@ func ReadJSON(r io.Reader, options ...LoadOption) DataFrame {
 	return LoadMaps(m, options...)
 }
 
+// WriteOption is the type used to configure the writing of elements
+type WriteOption func(*writeOptions)
+
+type writeOptions struct {
+	// Specifies whether the header is also written
+	writeHeader bool
+}
+
+// WriteHeader sets the writeHeader option for writeOptions.
+func WriteHeader(b bool) WriteOption {
+	return func(c *writeOptions) {
+		c.writeHeader = b
+	}
+}
+
 // WriteCSV writes the DataFrame to the given io.Writer as a CSV file.
-func (df DataFrame) WriteCSV(w io.Writer) error {
+func (df DataFrame) WriteCSV(w io.Writer, options ...WriteOption) error {
 	if df.Err != nil {
 		return df.Err
 	}
+
+	// Set the default write options
+	cfg := writeOptions{
+		writeHeader: true,
+	}
+
+	// Set any custom write options
+	for _, option := range options {
+		option(&cfg)
+	}
+
 	records := df.Records()
+	if !cfg.writeHeader {
+		records = records[1:]
+	}
+
 	return csv.NewWriter(w).WriteAll(records)
 }
 
 // WriteJSON writes the DataFrame to the given io.Writer as a JSON array.
-func (df DataFrame) WriteJSON(w io.Writer) error {
+func (df DataFrame) WriteJSON(w io.Writer, options ...WriteOption) error {
 	if df.Err != nil {
 		return df.Err
 	}
+
+	// Set the default write options
+	cfg := writeOptions{
+		writeHeader: true,
+	}
+
+	// Set any custom write options
+	for _, option := range options {
+		option(&cfg)
+	}
+
 	m := df.Maps()
+	if !cfg.writeHeader {
+		m = m[1:]
```

Comment:
Here you are omitting the first `map[string]interface{}` so you are removing the first row instead of removing the headers. There is no such thing as removing the headers from these maps.